server/.env.example

```
@@ -2,6 +2,9 @@ PORT=3001
 APP_BASE_URL="https://stellovault.com"
 DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/stellovault?schema=public"
 
+# SendGrid Configuration (for email notifications)
+SENDGRID_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)